redo.dev/docs/model/demos/stackblitzGenerators/rawDemoFiles.ts

```diff
@@ -0,0 +1,16 @@
+import constraints from "raw-loader!../static/generated/constraints.ts.raw"
+import declaration from "raw-loader!../static/generated/declaration.ts.raw"
+import group from "raw-loader!../static/generated/group.ts.raw"
+import model from "raw-loader!../static/generated/model.ts.raw"
+import space from "raw-loader!../static/generated/space.ts.raw"
+import user from "raw-loader!../static/generated/user.ts.raw"
+import { DeclarationFiles, EmbedId } from "./createStackblitzDemo"
+
+export const contentsByEmbedId: Record<EmbedId | DeclarationFiles, string> = {
```

Comment:
Let's split this into `contentsByEmbedId` and `contentsByAddonFile` so we don't accidentally try to use them interchangeably when they serve fundamentally different purposes in terms of their interaction with demo population.